lonboard/_cli.py

```diff
@@ -0,0 +1,108 @@
+import json
+import sys
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import click
+import pyarrow as pa
+import pyarrow.parquet as pq
+from pyogrio.raw import read_arrow
```

Comment:
because `pyogrio` is an optional dependency we should do something like 

```
try:
    import pyogrio
except ImportError:  # pragma: nocover
    pyogrio = None  # type: ignore
```

and at L93 raise an error with `pyogrio` is None

Reply:
I figured that I'd make it required for the CLI and be ok with an `ImportError`, but I suppose that we could allow the CLI to work for GeoParquet files without pyogrio installed